jdwp/common.py

```diff
@@ -0,0 +1,61 @@
+"""Basic types for JDWP messages."""
+
+from typing import List
+
+
+class Types:
+    """Types class."""
+
+    STRING = str
+    INT = int
+    BYTE = float
+    BOOLEAN = bool
+    DICT = dict
```

Comment:
This one seems odd

Reply:
Yeah, I've made modifications to the code. However, I have reservations about how we will accurately retrieve the specific datatype. Currently, I've implemented it as a string that holds the datatype's value. I'm uncertain if that works.